cmd/image-builder/main.go

```diff
@@ -28,6 +29,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func gitRev() string {
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		logrus.Warn("cannot read build info")
```

Comment:
(nipitck) I personally would avoid logrus in such a lowlevel helper and do that in the caller as I assume this code will be copied around/extracted into shared helpers and might be used in places where we use less logrus (bib for example). But feel free to ignore, it's very much a personal opinion and I see the benefits.

Reply:
good input, we could as well change the signature to return `string,error` I'll revise and reuse the function in composer, …